PixelHunter1995/Inputs/Input.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Input;
+
+namespace PixelHunter1995.Inputs
+{
+    using AllKeys = Either<Keys, MouseKeys>;
+
+    class Input : StateMap<AllKeys>
+    {
+
+        public Actions Actions { get; }
+
+        public int X { get; private set; }
+        public int Y { get; private set; }
+        public Point Position { get => new Point(this.X, this.Y); }
+
+        public int ScrollWheelValue { get; private set; }
+        public int HorizontalScrollWheelValue { get; private set; }
```

Comment:
Horizontal, really? Please remove.

Reply:
Depending on how the inventory is drawn, horizontal scrolling might make a lot of sense.
The actual reason it is there though is less to do with what it can be used for, and more to properly wrap all MouseState properties. With that in mind, should I remove it anyway?

Reply:
I think you should remove it.